conf/logback-test.xml

```diff
@@ -42,13 +42,13 @@
     </appender>
 
     <logger name="application" level="DEBUG"/>
-    <logger name="uk.gov" level="ERROR"/>
+    <logger name="uk.gov" level="DEBUG"/>
```

Comment:
Test logs already spit out much more than is necessary - why change to DEBUG?

Reply:
Not sure what you are asking... I have changed from ERROR to DEBUG